etna/whatson/models.py

```diff
@@ -0,0 +1,618 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+
+from modelcluster.fields import ParentalKey
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import RichTextField
+from wagtail.images import get_image_model_string
+from wagtail.models import Orderable
+from wagtail.search import index
+from wagtail.snippets.models import register_snippet
+
+from etna.articles.models import ArticleTagMixin
+from etna.collections.models import TopicalPageMixin
+from etna.core.models import BasePageWithIntro
+
+
+class BookingType(models.TextChoices):
+    """
+    This model is used for the Eventbrite booking type.
+    TODO: Find out what the other booking types are from Eventbrite.
+    """
+
+    DROP_IN = "drop_in", _("Drop in")
+    BOOKING = "booking", _("Booking")
+
+
+class VenueType(models.TextChoices):
+    """
+    This model is used to add venue types to event pages.
+    """
+
+    ONLINE = "online", _("Online")
+    IN_PERSON = "in_person", _("In person")
+    HYBRID = "hybrid", _("Hybrid")
+
+
+@register_snippet
+class EventType(models.Model):
+    """
+    This snippet model is used so that editors can add event types,
+    which we use via the event_type ForeignKey to add event types
+    to event pages.
+    """
+
+    name = models.CharField(
+        max_length=255,
+        verbose_name=_("name"),
+    )
+
+    slug = models.SlugField(
+        max_length=255,
+        verbose_name=_("slug"),
+    )
+
+    class Meta:
+        verbose_name = _("Event type")
+        verbose_name_plural = _("Event types")
```

Comment:
Django and Wagtail are pretty consistent in uppercasing the first character when needed, so there's no need to do that here. It's not explicitly mentioned in the Django docs to my knowledge, but there's a reason the examples for these aren't title-cased in the [reference docs](https://docs.djangoproject.com/en/4.2/ref/models/options/#label).

```suggestion
        verbose_name = _("event type")
        verbose_name_plural = _("event types")
```